sopel_modules/twitter/twitter.py

```diff
@@ -1,28 +1,43 @@
 # coding=utf-8
-from __future__ import unicode_literals, absolute_import, division, print_function
+from __future__ import annotations, unicode_literals, absolute_import, division, print_function
 
 from datetime import datetime
 import json
 import re
+from typing import TYPE_CHECKING
```

Comment:
Similarly to the above problem with trying to import `__future__.annotations`, the `typing` module does not exist before Python 3.5.

I appreciate that you want to use a type checker, but it's simply not possible to do in code intended for Sopel 7.x.